test/load/packer/consul-ami/consul.json

```diff
@@ -0,0 +1,67 @@
+{
+  "min_packer_version": "1.5.4",
+  "variables": {
+    "aws_region": "{{env `AWS_REGION`}}",
```

Comment:
If you are reading from the Terraform variable this should be `AWS_DEFAULT_REGION` when you make the change in the README.